py/desispec/io/raw.py

```diff
@@ -154,6 +154,14 @@ def read_raw(filename, camera, fibermapfile=None, **kwargs):
             loc = np.where(mod_fibers==fiber)[0]
             fibermap['FIBERSTATUS'][loc] |= maskbits.fibermask.BADFIBER
 
+    # Mask Fibers that are set to be excluded due to CCD/amp/readout issues
+    fibers_to_exclude = cfinder.fibers_to_exclude()
+    if fibers_to_exclude is not None:
```

Comment:
see comment above about `cfinder.fibers_to_exclude()` returning an empty list instead of None to slightly simplify logic here.